parse/EffectParser1.cpp

```diff
@@ -64,6 +66,8 @@ namespace {
             generate_sitrep_message
                 =    tok.GenerateSitrepMessage_
                 >    parse::label(Message_token)    >  tok.string [ _a = _1 ]
+                >  -(parse::label(Label_token)      >  tok.string [ _e = _1 ] )
+                >  -(tok.NoStringtableLookup_                     [ _f = true ] )
```

Comment:
@TheSilentOne1 changing the border color is fine with me if you want to take that on.  This last bit of polishing has taken far longer than I wanted it to & I'll not be adding any more bells and whistles myself anytime soon.